app/views/onboarding/consent/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="row justify-content-md-center">
+  <div class="col col-md-10 col-lg-8">
+    <h1>Data sharing permission</h1>
+    <p>To meet our requirements under the Smart Energy Code, please can you now give us formal permission for your school's energy data to be published publicly to the Energy Sparks website.<p>
+
+    <blockquote class="card bg-light">
+      <div class="card-body">
+        <p>I would like to enrol <strong><%=@school_onboarding.school_name%></strong> in the Energy Sparks service to allow the school to better monitor our energy usage.</p>
+        <p>I understand that in order for this to happen, our energy usage data will need to be published under an open licence to Energy Sparks. This includes both ongoing and historical data.</p>
+        <p>I understand that publishing the data under an open licence will allow it to be accessed, used and shared by anyone, including the Energy Sparks team.</p>
+        <p>I would like to give permission to begin publishing our energy usage data at the earliest opportunity.</p>
+        <p>I understand that our school has the right to withdraw consent to our data being published to Energy Sparks at anytime. I can withdraw consent by emailing hello@energysparks.uk. We will then remove your ongoing and historical data from the Energy Sparks site.</p>
+      </div>
+    </blockquote>
+
+    <%= button_to 'I give permission', onboarding_consent_path(@school_onboarding), method: :post, class: 'btn btn-primary' %>
```

Comment:
Maybe a back/cancel button or something? With the user knowing that they can return here without any probs.